packages/build/webpack/constants.js

```diff
@@ -236,13 +236,13 @@ module.exports = {
         ],
         TYPESCRIPT: [
             {
-                test: /^(?!.*spec\.tsx?$).*\.tsx?$/,
+                test: /\.(jsx|ts|tsx)(?!.*\.(spec|test)\.(jsx?|tsx?))$/,
```

Comment:
nit: The thing that is odd to me about this is the fact that `.jsx` files do not have TypeScript in them. I understand this is the minimal difference to get this working. It just seems like this makes things "unaligned" from a file processing perspective.

Reply:
The goal is for this to be a temporary update.